e2e/pages/events/enterRespondentsEvent.page.js

```diff
@@ -99,6 +103,16 @@ module.exports = {
     }
   },
 
+  async enterActiveParty(option) {
```

Comment:
If you don't mind having 2 methods you could make it clearer by having `setAsActiveParty()` and `setAsInactiveParty()`